generators/server/templates/src/test/java/package/web/rest/_AccountResourceIntTest.java

```diff
@@ -275,6 +275,9 @@ public void testGetUnknownAccount() throws Exception {
 
     @Before
     public void setup() {
+        <%_ if (databaseType === 'mongodb') { _%>
+        userRepository.deleteAll();
```

Comment:
mongodb doesn't do rollbacks for testing, which causes the `testRequestPasswordResetUpperCaseEmail` to fail